src/curl.cpp

```diff
@@ -4590,7 +4593,10 @@ void* S3fsMultiCurl::RequestPerformWrapper(void* arg)
 {
   S3fsCurl* s3fscurl= static_cast<S3fsCurl*>(arg);
   void*     result  = NULL;
-  if(s3fscurl && s3fscurl->fpLazySetup){
+  if(!s3fscurl){
+    return (void*)(intptr_t)(-EIO);
+  }
+  if(s3fscurl->fpLazySetup){
```

Comment:
Is this part of a separate change?

Reply:
My judgment may be wrong, but I got an error if I did not fix it like this with cppcheck 2.1.
I noticed an error when cppcheck2.1 may be installed in some cases.
(But this PR code uses cppcheck1.8x)

Reply:
I broken this into a separate PR #1329, preserving authorship.

Reply:
OK, Thanks.
I will remove this after merging #1329. And re-rebase this PR.